constants/upgradeSchema.json

```
@@ -28,9 +28,36 @@
     "troubleshootingUrl": "https://signoz.io/docs/user-guide/troubleshooting/"
   },
   "releases": {
+    "v0.92.0": {
+      "isMandatoryStop": true,
+      "nextMandatoryStop": null,
+      "releaseDate": "2025-08-13",
+      "patchRelease": {
+        "label": "v0.92.1",
+        "href": "https://github.com/SigNoz/signoz/releases/tag/v0.92.1"
+      },
+      "guideUrl": "https://signoz.io/docs/operate/migration/upgrade-0.92",
+      "instructions": [
+        "This release introduces the new Query Builder v5, migrates the CLI to a Cobra-based structure and support Clickhouse 25.5.6",
```

```suggestion
        "This release introduces Query Builder v5, migrates the CLI to a Cobra-based structure and supports Clickhouse 25.5.6",
```
Problematic grammar.